Problems creating new project

I have some problems regarding the used python versions when installing the backend.

I've setup the project with the settings below:

```
 peterm@Mac  ~/workspace  uvx cookieplone classic_project
╭─────────────────────────────── cookieplone ─────────────────────────────────╮
|                                                                             |
|                              .xxxxxxxxxxxxxx.                               |
|                          ;xxxxxxxxxxxxxxxxxxxxxx;                           |
|                       ;xxxxxxxxxxxxxxxxxxxxxxxxxxxx;                        |
|                     xxxxxxxxxx              xxxxxxxxxx                      |
|                   xxxxxxxx.                    .xxxxxxxx                    |
|                  xxxxxxx      xxxxxxx:            xxxxxxx                   |
|                :xxxxxx       xxxxxxxxxx             xxxxxx:                 |
|               :xxxxx+       xxxxxxxxxxx              +xxxxx:                |
|              .xxxxx.        :xxxxxxxxxx               .xxxxx.               |
|              xxxxx+          ;xxxxxxxx                 +xxxxx               |
|              xxxxx              +xx.                    xxxxx.              |
|             xxxxx:                      .xxxxxxxx       :xxxxx              |
|             xxxxx                      .xxxxxxxxxx       xxxxx              |
|             xxxxx                      xxxxxxxxxxx       xxxxx              |
|             xxxxx                      .xxxxxxxxxx       xxxxx              |
|             xxxxx:                      .xxxxxxxx       :xxxxx              |
|             .xxxxx              ;xx.       ...          xxxxx.              |
|              xxxxx+          :xxxxxxxx                 +xxxxx               |
|              .xxxxx.        :xxxxxxxxxx               .xxxxx.               |
|               :xxxxx+       xxxxxxxxxxx              ;xxxxx:                |
|                :xxxxxx       xxxxxxxxxx             xxxxxx:                 |
|                  xxxxxxx      xxxxxxx;            xxxxxxx                   |
|                   xxxxxxxx.                    .xxxxxxxx                    |
|                     xxxxxxxxxx              xxxxxxxxxx                      |
|                       ;xxxxxxxxxxxxxxxxxxxxxxxxxxxx+                        |
|                          ;xxxxxxxxxxxxxxxxxxxxxx;                           |
|                              .xxxxxxxxxxxxxx.                               |
|                                                                             |
╰─────────────────────────────────────────────────────────────────────────────╯
You've downloaded /Users/peterm/.cookiecutters/cookieplone-templates before. 
Is it okay to delete and re-download it? [y/n] (y):
╭────────────────────────────── Plone Project ────────────────────────────────╮
|                                                                             |
│ Creating a new Plone Project                                                |
|                                                                             |
│ Sanity check results:                                                       |
|                                                                             |
│   - Cookieplone: ✓                                                          |
│   - uv: ✓                                                                   |
│   - git: ✓                                                                  |
│   - Docker (optional): ✓                                                    |
|                                                                             |
╰─────────────────────────────────────────────────────────────────────────────╯
  [1/17] Project Title (Project Title):
  [2/17] Project Description (A new project using Plone 6.):
  [3/17] Project Slug (Used for repository id) (project-title):
  [4/17] Project URL (without protocol) (project-title.example.com):
  [5/17] Author (Plone Foundation):
  [6/17] Author E-mail (collective@plone.org):
  [7/17] Should we use prerelease versions? (No):
  [8/17] Plone Version (6.1.2):
  [9/17] Python Package Name (project.title):
  [10/17] Language
    1 - English
    2 - Deutsch
    3 - Español
    4 - Português (Brasil)
    5 - Nederlands
    6 - Suomi
    7 - Italiano
    8 - Svenska
    Choose from [1/2/3/4/5/6/7/8] (1):
  [11/17] GitHub or GitLab username or organization slug from URL (collective):
  [12/17] Container Registry
    1 - GitHub Container Registry
    2 - Docker Hub
    3 - GitLab
    Choose from [1/2/3] (1):
  [13/17] Which persistent storage to use in the deployment stack?
    1 - RelStorage with PostgreSQL (recommended)
    2 - ZEO with FileStorage
    3 - Local FileStorage, implies a single backend
    Choose from [1/2/3] (1): 3
  [14/17] Should we setup a caching server?
    1 - Yes
    2 - No
    Choose from [1/2] (1): 2
  [15/17] Add Ansible playbooks?
    1 - Yes
    2 - No
    Choose from [1/2] (1): 2
  [16/17] Add GitHub Action to Deploy this project?
    1 - Yes
    2 - No
    Choose from [1/2] (1): 2
  [17/17] Would you like to add a documentation scaffold to your project?
    1 - Yes
    2 - No
    Choose from [1/2] (1): 2
╭───────────────────────── Project Title generation ──────────────────────────╮
|                                                                             |
│ Summary:                                                                    |
|                                                                             |
│   - Plone version: 6.1.2                                                    |
│   - Output folder: /Users/peterm/workspace/project-title                    |
|                                                                             |
|                                                                             |
╰─────────────────────────────────────────────────────────────────────────────╯
 -> Setup Backend
 -> Ignoring (Generate documentation scaffold)
 -> Ignoring (Setup Cache)
 -> Classic Setup Project Settings
==> Format codebase
Installed 1 package in 4ms
Found 4 errors (4 fixed, 0 remaining).
Installed 1 package in 2ms
18 files left unchanged
Installed 6 packages in 23ms
 -> Remove Ansible files
 -> Remove GitHub Actions deployment files
 -> Remove unneeded documentation files
 -> Initialize Git repository
╭──────────────────────── New project was generated ──────────────────────────╮
|                                                                             |
│ Project Title                                                               |
|                                                                             |
│ Now, code it, create a git repository, push to your organization.           |
|                                                                             |
│ Sorry for the convenience,                                                  |
│ The Plone Community.                                                        |
|                                                                             |
│ https://plone.org/                                                          |
╰─────────────────────────────────────────────────────────────────────────────╯
```

When I install it, then the following error appears:

```
Install Backend
==> Generate constraints file
###############################################################################
# Load configuration
Can not parse override:
###############################################################################
# Read infiles
Read [r]: requirements.txt
Read [c]: https://dist.plone.org/release/6.1.2/constraints.txt
###############################################################################
# No sources configured!
###############################################################################
# Write outfiles
Write [c]: constraints-mxdev.txt
Write [r]: requirements-mxdev.txt
🎂 You are now ready for: pip install -r requirements-mxdev.txt
   (path to pip may vary dependent on your installation method)
==> Install environment
Using CPython 3.10.16
Creating virtual environment at: .venv
Activate with: source .venv/bin/activate
  × No solution found when resolving dependencies:
  ╰─▶ Because only project-title[test]==1.0.0a0 is available and the current Python version (3.10.16) does not satisfy Python>=3.12, we can conclude that all versions of project-title[test] cannot be used.
      And because you require project-title[test], we can conclude that your requirements are unsatisfiable.
make[2]: *** [/Users/peterm/workspace/project-title/backend/.venv] Error 1
make[1]: *** [backend-install] Error 2
make: *** [install] Error 2
```

You see, there's the line `Using CPython 3.10.6` but my python is setup with pyenv and uses `3.13.3`

```
peterm@Mac  ~/workspace/project-title   master ✚  python -V
Python 3.13.3
 peterm@Mac  ~/workspace/project-title   master ✚  python3 -V
Python 3.13.3
```

If i do a `uv python list` I get the following output:

```
cpython-3.14.0a7-macos-aarch64-none                 <download available>
cpython-3.14.0a7+freethreaded-macos-aarch64-none    <download available>
cpython-3.13.3-macos-aarch64-none                   /opt/homebrew/bin/python3.13 -> ../Cellar/python@3.13/3.13.3/bin/python3.13
cpython-3.13.3-macos-aarch64-none                   /opt/homebrew/bin/python3 -> ../Cellar/python@3.13/3.13.3/bin/python3
cpython-3.13.3-macos-aarch64-none                   /Users/peterm/.pyenv/shims/python3.13
cpython-3.13.3-macos-aarch64-none                   /Users/peterm/.pyenv/shims/python3
cpython-3.13.3-macos-aarch64-none                   /Users/peterm/.pyenv/shims/python
cpython-3.13.3-macos-aarch64-none                   <download available>
cpython-3.13.3+freethreaded-macos-aarch64-none      <download available>
cpython-3.12.10-macos-aarch64-none                  <download available>
cpython-3.12.9-macos-aarch64-none                   /opt/homebrew/bin/python3.12 -> ../Cellar/python@3.12/3.12.9/bin/python3.12
cpython-3.12.9-macos-aarch64-none                   /Users/peterm/.pyenv/shims/python3.12
cpython-3.11.12-macos-aarch64-none                  <download available>
cpython-3.11.11-macos-aarch64-none                  /opt/homebrew/bin/python3.11 -> ../Cellar/python@3.11/3.11.11/bin/python3.11
cpython-3.11.11-macos-aarch64-none                  /Users/peterm/.pyenv/shims/python3.11
cpython-3.10.17-macos-aarch64-none                  <download available>
cpython-3.10.16-macos-aarch64-none                  /Users/peterm/.local/share/uv/python/cpython-3.10.16-macos-aarch64-none/bin/python3.10
cpython-3.9.22-macos-aarch64-none                   <download available>
cpython-3.9.6-macos-aarch64-none                    /usr/bin/python3
cpython-3.8.20-macos-aarch64-none                   <download available>
pypy-3.11.11-macos-aarch64-none                     <download available>
pypy-3.10.16-macos-aarch64-none                     <download available>
pypy-3.9.19-macos-aarch64-none                      <download available>
pypy-3.8.16-macos-aarch64-none                      <download available>
graalpy-3.11.0-macos-aarch64-none                   <download available>
graalpy-3.10.0-macos-aarch64-none                   <download available>
graalpy-3.8.5-macos-aarch64-none                    <download available>
```

am I missing something to get the right python version for installing the backend?

--- a/news/254.bugfix
+++ b/news/254.bugfix
@@ -1,0 +1,1 @@
+Specify the Python version when creating a virtualenv in the backend Makefile. @davisagli

--- a/templates/add-ons/backend/{{ cookiecutter.__folder_name }}/Makefile
+++ b/templates/add-ons/backend/{{ cookiecutter.__folder_name }}/Makefile
@@ -53,7 +53,7 @@ requirements-mxdev.txt: pyproject.toml mx.ini ## Generate constraints file
 
 $(VENV_FOLDER): requirements-mxdev.txt ## Install dependencies
 	@echo "$(GREEN)==> Install environment$(RESET)"
-	@uv venv $(VENV_FOLDER)
+	@uv venv --python={{ cookiecutter.__supported_versions_python[0] }} $(VENV_FOLDER)
 	@uv pip install -r requirements-mxdev.txt
 
 .PHONY: sync

--- a/templates/sub/classic_project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile
+++ b/templates/sub/classic_project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile
@@ -53,7 +53,7 @@ requirements-mxdev.txt: pyproject.toml mx.ini ## Generate constraints file
 
 $(VENV_FOLDER): requirements-mxdev.txt ## Install dependencies
 	@echo "$(GREEN)==> Install environment$(RESET)"
-	@uv venv $(VENV_FOLDER)
+	@uv venv --python={{ cookiecutter.__supported_versions_python[0] }} $(VENV_FOLDER)
 	@uv pip install -r requirements-mxdev.txt
 
 .PHONY: sync

--- a/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile
+++ b/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile
@@ -53,7 +53,7 @@ requirements-mxdev.txt: pyproject.toml mx.ini ## Generate constraints file
 
 $(VENV_FOLDER): requirements-mxdev.txt ## Install dependencies
 	@echo "$(GREEN)==> Install environment$(RESET)"
-	@uv venv $(VENV_FOLDER)
+	@uv venv --python={{ cookiecutter.__supported_versions_python[0] }} $(VENV_FOLDER)
 	@uv pip install -r requirements-mxdev.txt
 
 .PHONY: sync

